### PR TITLE
Issue/quick 103 rebase

### PR DESCRIPTION
**Pull request by kema2000**

### CODE REVIEW

**Comment by dagguh** [comment]
@kema2000, please switch to https://github.com/atlassian/jira-hardware-exploration branching
Let's leave HWR forks behind, because Bamboo doesn't work with forks

**Comment by dagguh** [comment]
I'm not sure what's the intended change set here. Did you take a look at the commit list and the diff?
It seems like the only diff it brings is the custom setup password fix, which was already included in your other PR: https://github.com/atlassian/jira-hardware-exploration/pull/16

File: build.gradle.kts [hunk 1]
```diff
@@ -52,7 +52,7 @@ task<Test>("cleanUpAfterBamboo").apply {
 
 dependencies {
     testCompile(project(":virtual-users"))
-    compile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
+    testCompile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
```
Comment (atl-mk) [inline build.gradle.kts:55]: What's the difference between compile and testCompile?

Comment (atl-mk) [reply build.gradle.kts:55]: Found the answer here:https://github.com/atlassian/jira-hardware-exploration/pull/14/files#r272939312

**Comment by atl-mk** [comment]
> I'm not sure what's the intended change set here. Did you take a look at the commit list and the diff?
> It seems like the only diff it brings is the custom setup password fix, which was already included in your other PR: #16

This, I don't understand the impact of this refactoring (at least I can't see a functionality change)

**Pull request closed without merging**